golos.vesting/golos.vesting.hpp

```diff
@@ -11,20 +11,20 @@ namespace golos {
 
 using namespace eosio;
 
-class vesting : public contract {
+class [[eosio::contract("golos.vesting")]] vesting : public contract {
 public:
     using contract::contract;
 
-    [[eosio::action]] void validateprms(symbol symbol, std::vector<vesting_param>);
-    [[eosio::action]] void setparams(symbol symbol, std::vector<vesting_param>);
+    [[eosio::action]] void validateprms(symbol symbol, std::vector<vesting_param> params);
+    [[eosio::action]] void setparams(symbol symbol, std::vector<vesting_param> params);
 
     [[eosio::action]] void retire(asset quantity, name user);
     [[eosio::action]] void unlocklimit(name owner, asset quantity);
 
     [[eosio::action]] void withdraw(name from, name to, asset quantity);
-    [[eosio::action]] void stopwithdraw(name owner, symbol type);
-    [[eosio::action]] void delegate(name delegator, name delegatee, asset quantity, uint16_t interest_rate);
-    [[eosio::action]] void undelegate(name delegator, name delegatee, asset quantity);
+    [[eosio::action]] void stopwithdraw(name owner, symbol symbol);
+    [[eosio::action]] void delegate(name from, name to, asset quantity, uint16_t interest_rate);
+    [[eosio::action]] void undelegate(name from, name to, asset quantity);
```

Comment:
in case of `undelegate` `from` and `to` are confusing. (call of "undelegate from alice to bob" actually undelegates alices funds from bob). so here should be comment to clarify this in terms `delegator` and `delegatee`. same terms should be use in comment to `delegate`

Reply:
@zxcat In this case for what them used in manual ABI and how it bypassed review? :)

Reply:
who knows. I have huge questions for naming : )
but please add comments to fix this at least partly